RAMdbload_toFile.pl

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env perl
 use strict;
 use warnings;
-
+use Data::Dumper;
```

Comment:
Unless you use this somewhere else, I used Dumper for demonstration purposes and it isn't necessary.

Reply:
Oh I see, it is method of "dumping" the script variables to `sysout`